docs/source/onnxruntime/usage_guides/amdgpu.mdx

```diff
@@ -0,0 +1,117 @@
+# Accelerated inference on AMD GPUs
+
+By default, ONNX Runtime runs inference on CPU devices. However, it is possible to place supported operations on an AMD Instinct GPU, while leaving any unsupported ones on CPU. In most cases, this allows costly operations to be placed on GPU and significantly accelerate inference.
+
+This guide will show you how to run inference on the `ROCMExecutionProvider` execution provider that ONNX Runtime supports for AMD GPUs:
+
+## Installation
+
+#### 1. ROCM Installation (V 5.7.X)
+To install ROCM, please follow the [ROCM installation guide](https://rocm.docs.amd.com/en/latest/deploy/linux/index.html).
+
+#### 2. Torch Installation with ROCM Support
+
+```bash
+pip3 install --pre torch torchvision torchaudio --index-url https://download.pytorch.org/whl/nightly/rocm5.7
+```
+
+> **Note1:** Exporting with SDPA may encounter issues with torch versions below 2.1.0. To ensure smooth operation, consider upgrading to torch 2.1.0 or newer.
+
+> **Note2:** For docker installation, the following base image is recommended: rocm/pytorch:rocm5.7_ubuntu22.04_py3.10_pytorch_2.0.1
+
+### 3. ONNXRuntime Installation with ROCM Execution Provider
+
+```bash
+# pre-requisites
+pip install -U pip
+pip install cmake onnx
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+
+# Install ONNXRuntime from source
+git clone --recursive  https://github.com/ROCmSoftwarePlatform/onnxruntime.git
+git checkout rocm5.7_internal_testing_eigen-3.4.zip_hash
+cd onnxruntime
+
+./build.sh --config Release --build_wheel --update --build --parallel --cmake_extra_defines ONNXRUNTIME_VERSION=$(cat ./VERSION_NUMBER) --use_rocm --rocm_home=/opt/rocm
+pip install build/Linux/Release/dist/*
+```
+
+> **Note:** To avoid conflicts between `onnxruntime` and `onnxruntime-rocm`, make sure the package `onnxruntime` is not installed by running `pip uninstall onnxruntime` prior to installing `onnxruntime-rocm`.
```

Comment:
```suggestion
<Tip>
To avoid conflicts between `onnxruntime` and `onnxruntime-rocm`, make sure the package `onnxruntime` is not installed by running `pip uninstall onnxruntime` prior to installing `onnxruntime-rocm`.
</Tip>
```